build/depends.py

```diff
@@ -902,6 +902,7 @@ def sources(self, build):
                    "encoder/encodervorbis.cpp",
 
                    "tapfilter.cpp",
+                   "movinginterquartilemean.cpp",
```

Comment:
it looks like you added this file but forgot to add it to git. Please add this file and the corresponding header file. Then we can test your changes.